modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,34 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+[id="new-features-ossm-2-5-2"]
+== New features {SMProductName} version 2.5.2
+
+//Release is scheduled for May 22, 2024.
```

Comment:
Are the various date- and release-related comments important to keep in this PR?

Reply:
Yes in order to keep track of which release went out when since all releases are in a single adoc file.